src/reporter/coverage-report.xsl

```diff
@@ -117,7 +117,7 @@
             <p><span class="missed">not used</span></p>
          </xsl:when>
          <xsl:otherwise>
-            <pre>
+            <pre class="xspecCoverage">
```

Comment:
I added a class attribute to help the CSS recognize the whitespace class in a coverage report, as opposed to a test result report. Then, I also had to adjust the `xspec.bats` and `win-bats/collection.xml` files (Linux/macOS and Windows, respectively) because the test case that looks for the start of the `pre` element in a report file in a particular directory now sees the class attribute instead of the `<pre>` start tag without attributes.

Reply:
A couple of comments:

- does the browser need the help in the coverage report? As the coverage report doesn't have tables then it will not match the `td > pre > .whitespace`  selector and would just match a `.whitespace` selector.
- alternatively, would it be easier/better to stick with a single `.whitespace` selector that just sets the `color` value i.e. is there a benefit in making it the same as `.ignored` and `.comment`. The `font-style` doesn't matter and I think the `background` will be inherited from the `body` selector.  Or do you see any future benefit in separating the chosen selector between the test report and coverage report?

Reply:
What's complicated is that the test result report and the code coverage report use the same CSS file, but their content with `class="whitespace"` is not the same:

- In the test result report, `class="whitespace"` is for notation (like `\t` for a tab) that _visibly_ represents characters in a whitespace-only text node. The background color, font color, and font style matter.
- In the code coverage report, `class="whitespace"` is for literal whitespace characters.

It doesn't really matter visually if the coverage report picks up the test result report's whitespace style. But the other way around is a problem; it would be wrong for the test result report to use the coverage report's white background instead of pink vs. green, for whitespace notation in a diff. An example is [here](https://html-preview.github.io/?url=https://github.com/xspec/xspec/blob/master/test/end-to-end/cases/expected/stylesheet/serialize-result.html#top_scenario4) (search for "serialized with color" if your browser doesn't scroll to the right spot).

I thought about arranging the two whitespace CSS styles so that there is no harm done if the test result report happens to pick up the coverage report's style, based on the _existing_ style choices. And you are right that the coverage whitespace style doesn't have to match the ignored and comment styles. Then I decided that a careful arrangement was not as good as having the two whitespace styles have mutually exclusive CSS selectors. That makes it easier to change the CSS styles in the future without having unwanted interference between the two kinds of reports. Future changes aren't just theoretical because I do want to make some accessibility improvements.

Reply:
I didn't appreciate the way it is used in the test result report. 
So happy with your changes.

Reply:
I didn't notice the overlap at first, either.

Thanks for your review! I'll merge this change now.